src/StackExchange.Exceptional.AspNetCore/StackExchange.Exceptional.AspNetCore.csproj

```diff
@@ -9,7 +9,7 @@
   <ItemGroup>
     <ProjectReference Include="../StackExchange.Exceptional.Shared/StackExchange.Exceptional.Shared.csproj" />
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.0.0" />
```

Comment:
unfortunately, this is required in order to register the IHttpContextAccessor implementation:

https://github.com/m0sa/StackExchange.Exceptional/blob/73777dcd394dced02154842ecbae7a1f1b2da0c6/src/StackExchange.Exceptional.AspNetCore/ExceptionalServiceExtensions.cs#L42

Reply:
Let's be realistic, if you're using the library, this reference probably isn't *actually* adding one for anyone - I'm okay with it.